tests/test_conduit_cli.py

```diff
@@ -0,0 +1,179 @@
+
+
+@mock_ec2
+def test_is_task_running(mocked_cluster):
+    """Given an ECS Cluster ARN string, is_task_running should return True when
+    the task is running."""
+
+    # Create mocked ECS Cluster
+    mocked_ecs_client = boto3.client("ecs")
+    mocked_cluster_arn = mocked_cluster["cluster"]["clusterArn"]
+    # Create mocked EC2 instance
+    mocked_ec2_client = boto3.client("ec2")
+    mocked_ec2_images = mocked_ec2_client.describe_images(Owners=["amazon"])["Images"]
+    mocked_ec2_client.run_instances(ImageId=mocked_ec2_images[0]["ImageId"], MinCount=1, MaxCount=1)
+    mocked_ec2_instances = boto3.client("ec2").describe_instances()
+    mocked_ec2_instance_id = mocked_ec2_instances["Reservations"][0]["Instances"][0]["InstanceId"]
+    mocked_ec2 = boto3.resource("ec2")
+    mocked_ec2_instance = mocked_ec2.Instance(mocked_ec2_instance_id)
+    mocked_instance_id_document = json.dumps(ec2_utils.generate_instance_identity_document(mocked_ec2_instance))
+    # Attach mocked EC2 instance to the mocked ECS Cluster
+    mocked_ecs_client.register_container_instance(
+        cluster=mocked_cluster_arn, instanceIdentityDocument=mocked_instance_id_document
+    )
+    mocked_task_definition_arn = mocked_ecs_client.register_task_definition(
+        family="copilot-dbtunnel",
+        containerDefinitions=[
+            {"name": "test_container", "image": "test_image", "cpu": 100, "memory": 500, "essential": True}
+        ],
+    )["taskDefinition"]["taskDefinitionArn"]
+    mocked_ecs_client.run_task(cluster=mocked_cluster_arn, taskDefinition=mocked_task_definition_arn)
+
+    assert is_task_running(mocked_cluster_arn)
+
+
+@patch("os.system")
+@patch("commands.conduit_cli.is_task_running", return_value=True)
+def test_exec_into_task(is_task_running, system):
+    """Test that exec_into_task runs the `copilot task exec` command with
+    expected --app and --env flags."""
+
+    exec_into_task("dbt-app", "staging", "arn:random")
+
+    system.assert_called_once_with("copilot task exec --app dbt-app --env staging")
+
+
+@freeze_time("Jan 14th, 2020", auto_tick_seconds=60)
+def test_exec_into_task_timeout(capsys):
+    exec_into_task("dbt-app", "staging", "arn:random")
+
+    assert (
+        capsys.readouterr().out
+        == "Attempt to exec into running task timed out. Try again by running `copilot task exec --app dbt-app --env staging or check status of task in Amazon ECS console.\n"
+    )
+
+
+@mock_ecs
+@mock_resourcegroupstaggingapi
+@mock_sts
+def test_tunnel_no_cluster_resource(alias_session):
+    """Test that tunnel command prints an exit code when a cluster with app and
+    env tags is not found."""
+
+    runner = CliRunner()
+    result = runner.invoke(tunnel, ["--project-profile", "foo", "--app", "dbt-app", "--env", "staging"])
+
+    assert result.exit_code == 0
+    assert "No cluster resource found with tag filter values dbt-app and staging" in result.output
+
```

Comment:
How is the AWS profile `foo` set/configured for the test above, but not the one below?